timewarp_gl/shaders/timewarp_shader.hpp

```diff
@@ -1,7 +1,5 @@
 #include <GL/gl.h>
 
-#define GLSL_VERSION            "430 core"
-
 const char* const timeWarpChromaticVertexProgramGLSL =
 	"#version " GLSL_VERSION "\n"
```

Comment:
`GLSL_VERSION` macro is still being used in the shader source despite the `#define` being removed above?

Reply:
It's being defined at the command-line invocation of GCC. This way, it doesn't have to be duplicated in every shader.

Reply:
After seeing Finn's comment I'm wondering if putting the GLSL version in the makefile is the most readable choice. Perhaps it should be in an include of its own.

Reply:
If you do, put it in `common/`.